### ISSUE TITLE
Dividing tag break the other column tags in the description field.

### ISSUE THREAD

**Issue by Makpptfox** [stoneharry/Spell-Editor-GUI-V2#59]
Hi, this is a minor issue,

When you add a dividing tag (_$/10;s1_) for a certain effect, let's say effect1, the other effets won't be able to show the value in the "real time description" of the editor.

The dividing tag is used to show the final value of time or rage for exemple, since the input value is always bigger than the real final value the player have.

You can see it with the spell ID **17061**, the _$s2_ is not anymore able to show the value in real time, it does the same if you change the 2 by a 3, but if you remove the dividing tag, everything works again.

**Comment by stoneharry** [comment]
I didn't know this type of expression existed, yeah that's not currently supported. I will have a look at adding it in a couple of weeks.

**Comment by Makpptfox** [comment]
@stoneharry Did some tests, and it seems like there's a couple of other expressions the editor isn't able to interpret. Do you want me to propose a pullrequest on your tag documentations with all of the tags I know that aren't in it ?

Also, I did a full research on the 30 possibles misc tags used in SPELL_AURA_FLAT_MODIFIER(107) and SPELL_AURA_PCT_MODIFIER(108) if you want it as a documentation or as a list for a possible feature on your editor. (dropdown menu with those two specific Aura selected, if it's even possible)

**Comment by stoneharry** [comment]
@Makpptfox  Yes please! A pull request for updated doc would be brilliant. I will get around to working on this eventually, a few hours here and there.

**Comment by stoneharry** [comment]
I'm going to capture this additional formula using this regex: `\$(\/|\*|\-|)+\d+\;\d+\w.`, which would detect your example value: `$/10;17057s1`.

**Comment by stoneharry** [comment]
Added support for strings like `$/10;17057s1` in 3ef37b2.

![Parsed successfully](https://i.imgur.com/tpx5HhK.png)

I will use this issue to also track $? not being supported, as seen in spell id 66.

**Comment by stoneharry** [comment]
I did a parse on all spell string descriptions and tooltips in 3.3.5a enGB and collected all the ones that failed to parse. I have attached the results to this post.

It looks like pretty much all the main tags are handled now. The ones remaining are going to be difficult to handle but maybe one day I will get round to it.

[debug_unparsed_strings.txt](https://github.com/stoneharry/WoW-Spell-Editor/files/3801644/debug_unparsed_strings.txt)

**Comment by stoneharry** [comment]
Some work towards this done under #143
